docs/getting-started/use-cases/computervision/annotation/cvat/cvat_customizing_workspace_template.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Customizing CVAT Workspace Template
+sidebar_label: Environment variables - CVAT Workspace
```

Comment:
Change this to "Customizing CVAT Workspace Template"

Reply:
Agreed